aat/src/resources/features/F-018/F-018.feature

```diff
@@ -0,0 +1,62 @@
+@F-018
+Feature: F-018: CCD Case Activity Api :: Add Activity
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+
+  @S-001
+  Scenario: should invoke activity service And return response on successful requests
+    Given a user with [an active profile in CCD],
+    And a case that has just been created as in [Standard_Full_Case_Creation_Data],
+    When a request is prepared with appropriate values,
+    And the request [is to add an activity for the case created above],
+    And it is submitted to call the [ADD Activity] operation of [CCD Case Activity API],
+    Then a positive response is received,
+    And the response has all the details as expected,
+    And another call [to get the activity on the case above] will get the expected response as in [S-014-Verification-call].
```

Comment:
to observe the new activity just added on the case above

Reply:
updated